dji_sdk/src/modules/dji_sdk_node.cpp

```diff
@@ -458,8 +461,7 @@ DJISDKNode::initDataSubscribeFromFC(ros::NodeHandle& nh)
   topicList100Hz.push_back(Telemetry::TOPIC_ACCELERATION_GROUND);
   topicList100Hz.push_back(Telemetry::TOPIC_ANGULAR_RATE_FUSIONED);
 
-  int nTopic100Hz    = topicList100Hz.size();
-  if (vehicle->subscribe->initPackageFromTopicList(PACKAGE_ID_100HZ, nTopic100Hz,
+  if (vehicle->subscribe->initPackageFromTopicList(PACKAGE_ID_100HZ, static_cast<int>(topicList100Hz.size()),
```

Comment:
I think `static_cast` is unnecessary, as it will be converted implicitly to int.